### ISSUE TITLE
boundary missing for FIR RJJJ

### ISSUE THREAD

**Issue by dneuge** [vatsimnetwork/vatspy-data-project#325]
FIR `RJJJ` (Fukuoka) is defined in `VATSpy.dat` and referenced by multiple airports but has no entry in `FIRBoundaries.dat`:

```
RJJJ|Fukuoka|TOKYO|RJJJ
RJJJ|Fukuoka|TOKYOE|RJJJ
RJJJ|Fukuoka|TOKYOS|RJJJ
RJJJ|Fukuoka|TOKYOW|RJJJ
RJJJ|Fukuoka|TYO|RJJJ
RJJJ|Fukuoka|TYOE|RJJJ
RJJJ|Fukuoka|TYOS|RJJJ
RJJJ|Fukuoka|TYOW|RJJJ
```

This was also noticed during #230 but not fixed yet and was not tracked by any other issue yet.

**Comment by dsolesvik** [comment]
Looks like this is because the RJJJ FIR is on VATSIM split up into RJBG, RJCG, RJDG and RJTG ACCs. RJJJ_CTR and RJJJ_FSS have not been used for over 9 years according to VATSpy, so I believe it would probably be best to just remove the RJJJ entries and change the corresponding airport FIRs to the respective ACC.

**Comment by Adrian2k** [comment]
The best would having a staff member in Japan comment on this.

**Comment by ChrisXPP** [comment]
I'm no staff of VATSIM Japan, however I'd like to comment the "airport side" of this issue is fixed with PR #443 . Which means, no airport in the database is no longer "referenced" by RJJJ.
